packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -25,119 +25,73 @@ const AccountTransferBullet = ({ children }: React.PropsWithChildren) => (
-                        platform_name_dxtrade,
-                    }}
-                />
-            ) : (
-                <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv cryptocurrency and Deriv MT5 accounts.'
+                    i18n_default_text='We charge 2% or {{minimum_fee}} {{currency}} (whichever is higher) for all cryptocurrency transfers.'
                     values={{
                         minimum_fee,
                         currency: getCurrencyDisplayCode(currency),
                     }}
                 />
             );
-        } else if (transfer_fee === 2 && !is_mt_transfer && !is_dxtrade_transfer) {
+        } else if (transfer_fee === 1) {
             return (
                 <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv fiat and Deriv cryptocurrency accounts.'
+                    i18n_default_text='We charge 1% or {{minimum_fee}} {{currency}} (whichever is higher) for all cryptocurrency transfers.'
                     values={{
                         minimum_fee,
                         currency: getCurrencyDisplayCode(currency),
                     }}
                 />
             );
         }
-        return null;
-    }, [
-        currency,
-        is_crypto_to_crypto_transfer,
-        is_dxtrade_allowed,
-        is_dxtrade_transfer,
-        is_mt_transfer,
-        minimum_fee,
-        platform_name_dxtrade,
-        platform_name_mt5,
-        platform_name_ctrader,
-        transfer_fee,
-    ]);
+        return (
+            <Localize i18n_default_text='No fees for transfer between fiat account (with the same currency) to Deriv MT5, and Deriv X account(s), vice versa.' />
+        );
+    }, [currency, minimum_fee, transfer_fee]);
 
-    const getDxtradeAllowedNotes = useCallback(() => {
-        if (is_dxtrade_allowed) {
+    const getPlatformsAllowedNotes = useCallback(() => {
+        if (is_ctrader_transfer || is_dxtrade_transfer) {
             return (
                 <React.Fragment>
                     <AccountTransferBullet>
                         <Localize
-                            i18n_default_text='You may transfer between your Deriv fiat, cryptocurrency, {{platform_name_mt5}}, {{platform_name_ctrader}}, and {{platform_name_dxtrade}} accounts.'
-                            values={{ platform_name_dxtrade, platform_name_mt5, platform_name_ctrader }}
+                            i18n_default_text='Each day you can transfer up to {{ allowed_dxtrade }} {{ currency }}. The daily limit will be reset at 00:00 GMT.'
+                            values={{
+                                allowed_dxtrade: (exchange_rate * Number(allowed_transfers_amount?.dxtrade)).toFixed(2),
```

Comment:
Hi @lubega-deriv, Just a quick question, is there a chance for this to be not a number. Since, the value is coming from the client-store, there shouldn't be but don't know if this can happen 🤔.

Reply:
@aum-deriv good question 🤔 The fallback for `exchange_rate` is accounted for in cases where it's not available such as USD -> USD:
```const exchange_rate = exchange_rates?.USD?.[account_currency] || 1;```

But to be safe we could also add fallbacks for the transfer amounts based on their default values, thanks!